commands/help.go

```diff
@@ -60,223 +60,4 @@ var HelpCmd = &cli.Command{
-  GOLOG_FILE             Specify the name of the file that logs should be
-                         written to. Only used if GOLOG_OUTPUT contains the
-                         'file' keyword.
-
-  GOLOG_OUTPUT           Specify whether to output to file, stderr, stdout or
-                         a combination. Separate each keyword with a '+', for
-                         example: file+stderr
-
-  LILY_LOG_LEVEL_NAMED  Set the log level of specific loggers. The value
-                         should be a comma delimited list of log systems and
-                         log levels formatted as name:level, for example
-                         'logger1:debug,logger2:info'.
-
-In addition, lily supports some global options for controlling logging:
-
-  --log-level LEVEL        Set the default log level for all loggers to LEVEL.
-                           This option overrides any value set using the
-                           GOLOG_LOG_LEVEL environment variable.
-
-  --log-level-named value  Set the log level of specific loggers. This option
-                           overrides any value set using the
-                           LILY_LOG_LEVEL_NAMED environment variable.
-
-To control logging output while the lily daemon is running see 'visor help log'.
-
-During operation lily exposes metrics and debugging information on port 9991
-by default. The address used by this http server can be changed using the
-'--prometheus-port' option which expects an IP address and port number. The
-address may be omitted to run the server on all interfaces, for example: ':9991'.
-
-The following paths can be accessed using a standard web browser.
-
-  /metrics       Metrics published in prometheus format
-
-  /debug/pprof/  Access to standard Go profiling and debugging information
-                 memory allocations, cpu profile and active goroutines dumps.
-
-Visor can publish function level tracing to a Jaeger compatible service. By
-default tracing is disabled.
-
-Environment variables for controlling function level tracing:
-
-  LILY_TRACING         Enable tracing. Set to 'true' to enable tracing.
-
-  JAEGER_AGENT_HOST,    Hostname and port of a Jaeger compatible agent that
-  JAEGER_AGENT_PORT     lily should send traces to.
-
-  JAEGER_SERVICE_NAME   The name lily should use when reporting traces.
-
-  JAEGER_SAMPLER_TYPE,  Control the type of sampling used to capture traces.
-  JAEGER_SAMPLER_PARAM  The type may be either 'const' or 'probabilistic'.
-                        The behaviour of the sampler is controlled by the
-                        value of param. For a 'const' sampler a value of 1
-                        indicates that every function call should be traced,
-                        while 0 means none should be traced. No intermediate
-                        values are accepted. For a 'probabilistic' sampler
-                        the param indicates the fraction of function calls
-                        that should be sampled.
```

Comment:
I think a lot of these hidden flags and envvars are not documented anywhere. If this is true, could we see about including them somewhere appropriate?

Reply:
They are via:
```
./lily --help

<omitted some details>

GLOBAL OPTIONS:
   --api value                   Address of lily api in multiaddr format. (default: "/ip4/127.0.0.1/tcp/1231") [$LILY_API]
   --api-token value             Authentication token for lily api. [$LILY_API_TOKEN]
   --log-level LEVEL             Set the default log level for all loggers to LEVEL (default: "info") [$GOLOG_LOG_LEVEL]
   --log-level-named value       A comma delimited list of named loggers and log levels formatted as name:level, for example 'logger1:debug,logger2:info' [$LILY_LOG_LEVEL_NAMED]
   --jaeger-tracing              (default: false) [$LILY_JAEGER_TRACING]
   --jaeger-service-name value   (default: "lily") [$LILY_JAEGER_SERVICE_NAME]
   --jaeger-provider-url value   (default: "http://localhost:14268/api/traces") [$LILY_JAEGER_PROVIDER_URL]
   --jaeger-sampler-ratio value  If less than 1 probabilistic metrics will be used. (default: 1) [$LILY_JAEGER_SAMPLER_RATIO]
   --prometheus-port value       (default: ":9911") [$LILY_PROMETHEUS_PORT]
   --redis-addr value            Redis server address in "host:port" format (default: "127.0.0.1:6379") [$LILY_REDIS_ADDR]
   --redis-username value        Username to authenticate the current connection when redis ACLs are used. [$LILY_REDIS_USERNAME]
   --redis-password value        Password to authenticate the current connection [$LILY_REDIS_PASSWORD]
   --redis-db value              Redis DB to select after connection to server (default: 0) [$LILY_REDIS_DB]
   --version, -v                 print the version (default: false)
```